kvm-install-vm

```diff
@@ -618,11 +619,14 @@ _EOF_
         || die "Could not create storage pool."
 
     # Add custom MAC Address if specified
-    NETWORK_PARAMS="$(join ',' \
-        $(param bridge ${BRIDGE}) \
-        $(param model ${NETWORK_MODEL}) \
-        $(param mac ${MACADDRESS}) \
-        ${NETWORK_EXTRA})"
+    NETWORK_PARAMS=$(param network ${NETWORK_NAME})
```

Comment:
Ok, improved a little this behavior.
About quoting "param" funtion parameters, I did not because isn't the script style. I think this should be done in a separate PR

Reply:
It is. That's true. I have a local branch to update quotes and other such changes that I haven't pushed yet. It's OK if it's left out of this PR. I was trying to enforce proper styling on each new PR.